### PR TITLE
Replaces device_role with role on device model

### PR DESCRIPTION
**Pull request by abhi1693**

### CODE REVIEW

File: netbox/dcim/tables/devices.py [hunk 1]
```diff
@@ -285,14 +285,14 @@ class DeviceTable(TenancyColumnsMixin, ContactsColumnMixin, NetBoxTable):
     class Meta(NetBoxTable.Meta):
         model = models.Device
         fields = (
-            'pk', 'id', 'name', 'status', 'tenant', 'tenant_group', 'device_role', 'manufacturer', 'device_type',
+            'pk', 'id', 'name', 'status', 'tenant', 'tenant_group', 'role', 'manufacturer', 'device_type',
```
Comment (jeremystretch) [inline netbox/dcim/tables/devices.py:288]: Renaming the column might cause an issue with users' saved table preferences. We may be able to address this in the database migration, updating any occurrence of `device_role` in `tables.DeviceTable.columns`.

File: netbox/netbox/settings.py [hunk 2]
```diff
@@ -469,7 +469,7 @@ def _setting(name, default=None):
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
-TEST_RUNNER = "django_rich.test.RichRunner"
+# TEST_RUNNER = "django_rich.test.RichRunner"
```
Comment (jeremystretch) [inline netbox/netbox/settings.py:472]: Guessing this was intended to be a temporary change? (Happy to make the test runner configurable.)

**Review by jeremystretch — CHANGES_REQUESTED** [review]
Thanks for plowing through all of this!

File: netbox/netbox/settings.py (hunk 2 above)
Comment (abhi1693) [reply netbox/netbox/settings.py:472]: Yeah, I do not like the colored output. Makes it difficult to find the issue. I'll revert this.

File: netbox/dcim/tables/devices.py (hunk 1 above)
Comment (jeremystretch) [reply netbox/dcim/tables/devices.py:288]: I've added a routine in the migration file to update references to the old `device_role` table column in user configs.

**Pull request merged**